src/gatsby/gatsby-config.js

```diff
@@ -171,6 +182,7 @@ module.exports = {
                 protectedArea: {
                   fields: "*"
                 },
+                // TODO: site doesn't have a relation with park-feature yet
```

Comment:
There actually is a relation, but it's incorrect. It's `oneToOne` and it should be `manyToOne`. We should probably fix this and release it with 2.14.2

```
    "site": {
      "type": "relation",
      "relation": "oneToOne",
      "target": "api::site.site"
    },
```